This test is used to test other errors instead of TimeoutError,
according to the attached comment, increase timeout should be fine.
I run into the timeout 1 out of 10 run.
diff --git nbconvert/preprocessors/tests/test_execute.py nbconvert/preprocessors/tests/test_execute.py
index 243a725..a55116e 100644
--- nbconvert/preprocessors/tests/test_execute.py
+++ nbconvert/preprocessors/tests/test_execute.py
@@ -317,7 +317,7 @@ def test_many_parallel_notebooks(capfd):
     with ProcessPool(max_workers=4) as pool:
         futures = [
             # Travis needs a lot more time even though 10s is enough on most dev machines
-            pool.schedule(run_notebook, args=(input_file, opts, res), timeout=30)
+            pool.schedule(run_notebook, args=(input_file, opts, res), timeout=60)
             for i in range(0, 8)
         ]
         for index, future in enumerate(futures):
